automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -2,48 +2,78 @@ package org.broadinstitute.dsde.workbench.leonardo
 
 import java.io.File
 import java.nio.file.Files
-import java.time.Instant
 
+import com.typesafe.scalalogging.LazyLogging
 import org.broadinstitute.dsde.firecloud.api.{Orchestration, Rawls, Sam}
+import org.broadinstitute.dsde.workbench.WebBrowserSpec
 import org.broadinstitute.dsde.workbench.api.APIException
-import org.broadinstitute.dsde.workbench.{ResourceFile, WebBrowserSpec}
 import org.broadinstitute.dsde.workbench.config.{AuthToken, WorkbenchConfig}
+import org.broadinstitute.dsde.workbench.dao.Google.googleIamDAO
 import org.broadinstitute.dsde.workbench.leonardo.ClusterStatus.ClusterStatus
 import org.broadinstitute.dsde.workbench.leonardo.StringValueClass.LabelMap
-import org.broadinstitute.dsde.workbench.util.LocalFileUtil
-import org.broadinstitute.dsde.workbench.dao.Google.googleIamDAO
 import org.broadinstitute.dsde.workbench.model.WorkbenchEmail
 import org.broadinstitute.dsde.workbench.model.google.{GoogleProject, ServiceAccountName}
+import org.broadinstitute.dsde.workbench.util.LocalFileUtil
 import org.openqa.selenium.WebDriver
+import org.scalatest.{Matchers, Suite}
 import org.scalatest.concurrent.{Eventually, ScalaFutures}
-import org.scalatest.selenium.WebBrowser
 import org.scalatest.time.{Minutes, Seconds, Span}
-import org.scalatest.{FreeSpec, Matchers, ParallelTestExecution}
 
-import scala.util.control.NonFatal
 import scala.util.{Random, Try}
+import scala.util.control.NonFatal
 
-class LeonardoSpec extends FreeSpec with Matchers with Eventually with ParallelTestExecution
-  with WebBrowser with WebBrowserSpec with LocalFileUtil with ScalaFutures {
+trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually with LocalFileUtil with LazyLogging with ScalaFutures {
+  this: Suite =>
 
-  implicit override val patienceConfig: PatienceConfig = PatienceConfig(timeout = scaled(Span(5, Seconds)))
+  val swatTestBucket = "gs://leonardo-swat-test-bucket-do-not-delete"
+  val incorrectJupyterExtensionUri = swatTestBucket + "/"
 
   // Ron and Hermione are on the dev Leo whitelist.
   val ronAuthToken = AuthToken(LeonardoConfig.Users.ron)
   val hermioneAuthToken = AuthToken(LeonardoConfig.Users.hermione)
 
-  // kudos to whoever named this Patience
   val clusterPatience = PatienceConfig(timeout = scaled(Span(15, Minutes)), interval = scaled(Span(20, Seconds)))
   val localizePatience = PatienceConfig(timeout = scaled(Span(1, Minutes)), interval = scaled(Span(1, Seconds)))
 
-  val sa = GoogleServiceAccount(LeonardoConfig.Leonardo.notebooksServiceAccountEmail)
-  val swatTestBucket = "gs://leonardo-swat-test-bucket-do-not-delete"
-  val incorrectJupyterExtensionUri = swatTestBucket + "/"
+  // TODO: show diffs as screenshot or other test output?
```

Comment:
I wonder `listA shouldEqual listB` would display the diff in the test logs. Agreed that would be useful